src/features/pages/conversational-learning-page.html

```diff
@@ -0,0 +1,462 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Collaboration and Contribution</title>
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+  <link href="https://fonts.googleapis.com/css2?family=DM+Sans:opsz@9..40&display=swap" rel="stylesheet">
+  <style>
+
+body {
+    background: #FAE9E8 0% 0% no-repeat padding-box;
+    opacity: 1;
+    font-family: DM Sans;
+    }
+
+.main1 {
+    display: flex;
+    flex-direction: column;
+    margin: 2% 4%;
+}
+
+h1, h2, p, ul, li{
+    font-family: 'DM Sans', sans-serif;
+}
+
+.heading {
+    left: 209px;
+    width: 839px;
+    height: 106px;
+    text-align: left;
+    font: normal normal  82px/110px DM Sans;
+    letter-spacing: 0px;
+    color: #000000;
+    opacity: 1;
+
+}
+h3 {
+    margin-top: 0;
+    left: 209px;
+    width: 701px;
+    height: 224px;
+    text-align: left;
+    font: normal normal normal 32px/42px DM Sans;
+    letter-spacing: 0px;
+    color: #000000;
+    opacity: 1;
```

Comment:
here and rest of the style

Reply:
@JAPHETHNYARANGA  i have made the changes requested. I request a review please.